docs/antrea-agent-simulator.md

```diff
@@ -0,0 +1,62 @@
+# Run Antrea agent simulator
+
+This Documentation describes how to run Antrea agent simulator.
+
+## Build the images
+
+1. Build simulator image
+
+   ```bash
+   make build-scale-simulator
+   ```
+
+2. Build the kubemark image
+
+If you alreay have a kubemark image, please ignore this step.
```

Comment:
Need to keep align with "Build the kubemark image", otherwise it wouldn't appear as part of step 2.